### ISSUE TITLE
Keyboard passthrough not working on Apple TV

### ISSUE THREAD

**Issue by bverkron** [moonlight-stream/moonlight-ios#434]
Paired a bluttooth keyboard with my Apple TV. Works on the home screen and in the moonlight main menu but it's not passed through to the PC I'm connected to. Nothing happens when I type or use hotkeys, etc. Tried on the login screen and in various apps when launched into mstsc for full desktop streaming.

**Comment by hepto** [comment]
I actually have this issue but with my DualShock 4 controller, wonder if it is the same.  On the host PC the Virtual USB devices are installed, but no input on the controller seems to be passed through to the host PC even though it works on the Moonlight navigation itself.

**Comment by cgutman** [comment]
@bverkron Thanks for the report. This is a regression in v7.0.0. I've submitted v7.0.1 to Apple for review.

@hepto I don't think it's the same issue. Controller input is totally different code. Usually controller problems are related to software running on your PC like Steam controller configurator. You can join our Discord server to get someone to help figure out what is breaking controller input.

**Comment by bverkron** [comment]
Fantastic, thank you! Must say it's been a while since I tried moonlight and was pleasantly surprised when I found it was now available for tvOS. I have to move my gaming PC from the living room to my office space and this is a great fit for streaming to my existing Apple TV 4K in the living room. Works beautifully and so much nicer than Steam for lots of reasons but especially because I can unlock my PC remotely right in moonlight!
